blazar/manager/service.py

```diff
@@ -448,9 +477,7 @@ def update_lease(self, lease_id, values):
                     pass
                 resource_type = v.get('resource_type',
                                       reservation['resource_type'])
-                if resource_type != reservation['resource_type']:
```

Comment:
It feels like this should stay in the code - it checks that a reservation doesn't suddenly change types, which should be unsupported I think. What do you think?